### DISCUSSION TITLE
Copilot crash alaways in visual studio 2022

### DISCUSSION THREAD

**Discussion by linjiahao962889027** [community/community#38738] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/46292984/200975952-cb9bbe54-fdea-4ce7-ad15-adab6063a202.png)
![image](https://user-images.githubusercontent.com/46292984/200975994-43071771-9846-4c81-8f60-01100b03dc5c.png)
In visual studio 2022(17.4)，the copilot is alaways crash.

**Comment by iamcjwilliams** [comment]
Hello @linjiahao962889027 please try the steps below:

1. Close Visual Studio 2022.
2. Delete C:\Users<insert username>\AppData\Local\github-copilot\hosts.json
3. Log out of GitHub in the web browser.
4. Open Visual Studio.

**Reply by linjiahao962889027** [reply]
It can’t address this problem.

**Reply by cmp-nct** [reply]
Noticing your Mandarin/chinese characters, you are likely behind a "great fire-wall" ?
I recommend trying to use a VPN that works in your area so your network traffic is certainly not attacked by MIM or something similar.

**Reply by linjiahao962889027** [reply]
> Noticing your Mandarin/chinese characters, you are likely behind a "great fire-wall" ? I recommend trying to use a VPN that works in your area so your network traffic is certainly not attacked by MIM or something similar.

No, the copilot works well on the Visual Studio Code and IntelliJ IDEA. I've been using copilot more than 1000 hours on the Visual Studio Code, so I think the problem is from the copilot's plugin of Visual Studio. If the serve support of copilot have different ip, it may be the problem of network.

**Comment by cmp-nct** [comment]
The Visual Studio Plugin is very unstable, I recommend using another IDE.
The Jetbrain Plugin is also very minimalistic but at least it's stable.
I've been using copilot for probably 150 hours of coding by now, it's an extreme pain on Visual Studio.
